src/ResolvedName.re

```diff
@@ -95,24 +95,42 @@ let rec extendExportModules = (x, ~exportModuleItems, ~valueName) =>
     }
   };
 
-let globalExportModuleItems: exportModuleItems = Hashtbl.create(1);
+type moduleItemsEmitter = Hashtbl.t(string, exportModuleItem);
+
+let createModuleItemsEmitter = () => Hashtbl.create(1);
+
+let emitAllModuleItems = (~moduleItemsEmitter) => {
+  let items = ref([]);
```

Comment:
is there a better way to insert those tuple into an array, because we iter on the map